Spec/API.md

```diff
@@ -373,6 +374,7 @@ Response description:
 - `last_update`: ISO format UTC datetime string describing the time of the most recent deck update
 - `last_update_device`: the name of the device that last modified the deck (this is only available for decks that are owned by the requester)
 - `share_code`: the code that can be entered by users to gain access to a remote private deck
+- `deleted`: boolean indicating whether or not a deck has been deleted by the owner
```

Comment:
If `deleted` is true, should the client still expect to receive all the other metadata?